test/verify/check-openshift

```diff
@@ -391,542 +391,6 @@ LABEL io.projectatomic.nulecule.atomicappversion="0.1.11" \
         b.click("#kubernetes-reconnect")
         b.wait_present("#service-list")
 
-
-class TestOpenshiftPrerelease(TestOpenshift):
-    provision = {
-        "machine1": { "address": "10.111.113.1/20" },
-        "openshift": { "image": "openshift-prerelease", "forward": { 30000: 30000, 8443: 8443 } }
-    }
```

Comment:
This part needs to stay - we still want to run the non-kubevirt TestOpenshiftPrerelease tests.

Reply:
Oh totally missed that TestOpenshiftPrerelease inherited TestOpenshift, now I understand where those tests come from :D